l10n_hr_account_base/models/account_move.py

```diff
@@ -63,6 +63,18 @@ class AccountMove(models.Model):
         " like 2 or more devices for this journal",
     )
 
+    l10n_hr_tax_note = fields.Char(string='tax field')
+    
+    def action_post(self):
+        res = super().action_post()
+        tax_notes = ''
+        for line in self.invoice_line_ids:
+            for tax_id in line.tax_ids:
```

Comment:
@lvrabec 
ovo je krivo jer bi trebao uzeti sve poreze pa sa njih procitati opise te paziti da ne dodas vise puta opis za isti porez jer sada to radis